### PR TITLE
feat(workflow): Add beta badge and warning to session % rule

### PR DESCRIPTION
**Pull request by scttcper**
Beta badge in dropdown
Screen Shot 2021-06-29 at 4 21 37 PM

Inline beta badge. 
Re-styled inline warnings and added a warning for session%
Screen Shot 2021-06-29 at 5 15 09 PM

Add stories for FeatureBadge

### CODE REVIEW

File: docs-ui/components/featureBadge.stories.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
```
Comment (EvanPurkhiser) [inline docs-ui/components/featureBadge.stories.js:1]: would be nice as it's own PR 😎

Comment (scttcper) [reply docs-ui/components/featureBadge.stories.js:1]: no this is how i sneak things in

Comment (EvanPurkhiser) [reply docs-ui/components/featureBadge.stories.js:1]: yeah but I would have already approved this as a separate PR :trollface:

File: static/app/views/settings/projectAlerts/issueRuleEditor/ruleNode.tsx [hunk 2]
```diff
@@ -415,3 +419,7 @@ const DeleteButton = styled(Button)`
 const MarginlessAlert = styled(Alert)`
   margin: 0;
 `;
+
+const StyledFeatureBadge = styled(FeatureBadge)`
+  margin: 0 ${space(1)} 0 0;
```
Comment (EvanPurkhiser) [inline static/app/views/settings/projectAlerts/issueRuleEditor/ruleNode.tsx:424]: Can you just use `margin-right`?

Comment (scttcper) [reply static/app/views/settings/projectAlerts/issueRuleEditor/ruleNode.tsx:440]: in this case i also wanted to override the margin as well

Comment (EvanPurkhiser) [reply static/app/views/settings/projectAlerts/issueRuleEditor/ruleNode.tsx:440]: Ah it has a default? ew

Comment (scttcper) [reply static/app/views/settings/projectAlerts/issueRuleEditor/ruleNode.tsx:440]: very ew, most of our badges dont

**Comment by scttcper** [comment]
how the warning looked before changes
image

**Review by taylangocmen — APPROVED** [review]
